_docs/bitrise-yml-format-spec.md

```diff
@@ -36,6 +36,7 @@ workflows:
 ## App properties
 
 - `envs` : configuration global environment variables list
+- `bitrise.io.stack` : the default stack for the given project
```

Comment:
`meta` is the new property and it can store any map[string]interface{} value, we will use it for storing the default stack in the Bitrise.yml, but it is a much more general-purpose field

Reply:
Updated readme.